src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -27,6 +28,14 @@ public static partial class RoslynInsertionTool
         public static readonly Guid MLInfraSwatUserId = new Guid("6c25b447-1d90-4840-8fde-d8b22cb8733e");
         public static readonly Guid VSLSnapUserId = new Guid("9f64bc2f-479b-429f-a665-fec80e130b1f");
 
+        /// <summary>
+        /// Dnceng maintain and build from a copy of dotnet/roslyn. When trying to get the build from dnceng, it will
+        /// only return the id of the copy, not the real dotnet/roslyn in GitHub.
+        /// Therefore, we need this to map back to Roslyn
+        /// </summary>
+        public static readonly ImmutableDictionary<string, string> dncengRepoNameToGitHubId = ImmutableDictionary<string, string>
```

Comment:
This is assuming whenever friend repos also moved to dnceng, they also need to add a value here